README.template

```diff
@@ -47,6 +47,9 @@ services:
 
 Then run all services `docker-compose up -d`. Now, go to http://0.0.0.0 to access to the new Dolibarr installation.
 
+## Upgrading version and migrating DB
+Remove the `install.lock` file and restart container. It will migrate Database to the new version.
```

Comment:
```suggestion
Remove the `install.lock` file and start a updated version container. Ensure that env `DOLI_INSTALL_AUTO` is set to `1`. It will migrate Database to the new version.
```